eemeter/ee/meter.py

```diff
@@ -482,782 +524,60 @@ def evaluate(self, meter_input, formatter=None,
-                        ],
-                        'variance': [
-                            0 if not reporting_mask.get(i, True) else None
-                            for i, v in unmasked_reporting_period_daily_fixture['tempF'].iteritems()
-                        ],
-                    })
-                except:
-                    _report_failed_derivative(series)
-
-            if weather_normal_source_success:
-                series = 'Temperature, normal year'
-                description = '''Observed temperature (degF) over the normal year.'''
-                try:
-                    raw_derivatives.append({
-                        'series': series,
-                        'description': description,
-                        'orderable': [
-                            i.isoformat() for i in annualized_daily_fixture.index],
-                        'value': annualized_daily_fixture['tempF'].values.tolist(),
-                        'variance': [0 for _ in range(annualized_daily_fixture['tempF'].shape[0])]
-                    })
-                except:
-                    _report_failed_derivative(series)
-
-            series = 'Inclusion mask, baseline period'
-            description = '''Mask for baseline period data which is included in
-                             model and savings cumulatives.'''
-            try:
-                raw_derivatives.append({
-                    'series': series,
-                    'description': description,
-                    'orderable': [i.isoformat() for i in baseline_mask.index],
-                    'value': [bool(v) for v in baseline_mask.values],
-                    'variance': [0 for _ in range(baseline_mask.shape[0])]
-                })
-            except:
-                _report_failed_derivative(series)
-
-            series = 'Inclusion mask, reporting period'
-            description = '''Mask for reporting period data which is included in
-                             model and savings cumulatives.'''
-            try:
-                raw_derivatives.append({
-                    'series': series,
-                    'description': description,
-                    'orderable': [i.isoformat() for i in reporting_mask.index],
-                    'value': [bool(v) for v in reporting_mask.values],
-                    'variance': [0 for _ in range(reporting_mask.shape[0])]
-                })
-            except:
-                _report_failed_derivative(series)
+            deriv_input = unpack(modeled_trace, baseline_label, reporting_label,
+                                 baseline_period, reporting_period, 
+                                 weather_source, weather_normal_source,
+                                 derivative_freq=derivative_freq)
+            if deriv_input is None:
+                continue
+            raw_derivatives.append(hdd_balance_point_baseline(deriv_input))
```

Comment:
Minor comment for slightly more readable code, since all these long lines get hard to read. Instead of all these appends, you could just extend one list, i.e.,

```
raw_derivatives.extend([
    hdd_balance_point_baseline(deriv_input),
    hdd_coefficient_baseline(deriv_input),
    cdd_balance_point_baseline(deriv_input),
    ...
])
```

or even this, as long as it doesn't obfuscate too much

```
derivative_methods = [
[
    hdd_balance_point_baseline,
    hdd_coefficient_baseline,
    cdd_balance_point_baseline,
    ...
]
raw_derivatives.extend([
    derivative_method(deriv_input)
    for derivative_method in derivative_methods
])
```

Totally fine to leave it as is as well.